src/bitmessageqt/account.py

```diff
@@ -1,26 +1,58 @@
-from PyQt4 import QtCore, QtGui
+# pylint: disable=too-many-instance-attributes,attribute-defined-outside-init
+"""
+account.py
+==========
 
-import queues
+Account related functions.
+
+"""
+
+from __future__ import absolute_import
+
+import inspect
 import re
 import sys
-import inspect
-from helper_sql import *
-from helper_ackPayload import genAckPayload
+import time
+
+from PyQt4 import QtGui
+
+import queues
 from addresses import decodeAddress
 from bmconfigparser import BMConfigParser
-from foldertree import AccountMixin
-from pyelliptic.openssl import OpenSSL
-from utils import str_broadcast_subscribers
-import time
+from helper_ackPayload import genAckPayload
+from helper_sql import sqlQuery, sqlExecute
+from .foldertree import AccountMixin
+from .utils import str_broadcast_subscribers
+
 
 def getSortedAccounts():
+    """Get a sorted list of configSections"""
+
     configSections = BMConfigParser().addresses()
-    configSections.sort(cmp = 
-        lambda x,y: cmp(unicode(BMConfigParser().get(x, 'label'), 'utf-8').lower(), unicode(BMConfigParser().get(y, 'label'), 'utf-8').lower())
-        )
+    configSections.sort(
+        cmp=lambda x, y: cmp(
+            unicode(
+                BMConfigParser().get(
+                    x,
+                    'label'),
+                'utf-8').lower(),
+            unicode(
+                BMConfigParser().get(
+                    y,
+                    'label'),
+                'utf-8').lower()))
```

Comment:
Are you joking?

Reply:
Guessing you don't like the formatting? I was just following pylint's indentation. The whole BMConfigParser is a copy-pasta boiler-plate abomination that needs ripping out at some point. If you confirm that this style is your issue I'll be happy to change it, as usual.